app/src/androidTest/java/org/mozilla/tv/firefox/helpers/TestAssetHelper.kt

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.helpers
+
+import android.net.Uri
+import okhttp3.mockwebserver.MockWebServer
+import org.mozilla.tv.firefox.ext.toUri
+
+object TestAssetHelper {
+
+    data class TestAsset(val url: Uri, val urlStr: String, val content: String)
+
+    fun getGenericAssets(server: MockWebServer): List<TestAsset> {
```

Comment:
I think that's a cool idea, but I'm not sure the cost / benefit pencils out in this case.  Do you think this would be worth talking over at a team meeting?

Reply:
Ah, nah – it's not very discoverable and people probably won't use it. Let's wait until it's actually a problem before we do that.